### DISCUSSION TITLE
Self control not working at all

### DISCUSSION THREAD

**Discussion by merryheart666** [SelfControlApp/selfcontrol#761] [Q&A]
I just downloaded SelfControl 4.0.2 (401) to MacBook Pro running Monterey 12.0.01 and made a whitelist for just 3 websites. Using Firefox 94.0.2. No blocking taking place. I cleared cache and restarted, still not working. Firefox set to No Proxy. Any help appreciated Tx!!

**Comment by rixj** [comment]
Still not working on 10.15.7 after restarting browsers and computer.

**Comment by cstigler** [comment]
Hi all, sorry it's not working! Unfortunately, the reasons it might not be blocking vary from situation-to-situation, computer-to-computer, and network-to-network. The best troubleshooting tips I know are still in this document: https://github.com/SelfControlApp/selfcontrol/wiki/SelfControl-Isn't-Blocking-Sites,-What-Do-I-Do%3F
